pkgs/top-level/all-packages.nix

```diff
@@ -23950,7 +23950,7 @@ in
 
   kooha = callPackage ../applications/video/kooha { };
 
-  kotatogram-desktop = libsForQt514.callPackage ../applications/networking/instant-messengers/telegram/kotatogram-desktop { };
+  kotatogram-desktop = qt5.callPackage ../applications/networking/instant-messengers/telegram/kotatogram-desktop { };
```

Comment:
```suggestion
  kotatogram-desktop = libsForQt5.callPackage ../applications/networking/instant-messengers/telegram/kotatogram-desktop { };
```

Reply:
I'm not sure whether this is is correct. This derivation is almost a copy of the tdesktop's derivation and it uses qt5.callPackage.

Reply:
libsForQt5 allows changing the qt version used. I don't think we need this here but otherwise it is the same.

Reply:
Ok, changed